### ISSUE TITLE
Test suite failures on Strawberry Perl

### ISSUE THREAD

**Issue by plicease** [wbraswell/alien-astyle#4]
```
N:\home\ollisg\dev\Alien-astyle>prove -bv t\02_binary_executable.t
t\02_binary_executable.t ..
1..10
ok 1 - use Alien::astyle;
ok 2 - Alien::astyle->bin_dir() is defined
ok 3 - Alien::astyle->bin_dir() is not empty

ok 4 - Alien::astyle->bin_dir() exists
#   Failed test 'astyle exists'
ok 5 - Alien::astyle->bin_dir() is readable
#   at t\02_binary_executable.t line 18.
ok 6 - Alien::astyle->bin_dir() is a directory

not ok 7 - astyle exists
#   Failed test 'astyle is readable'
not ok 8 - astyle is readable
#   at t\02_binary_executable.t line 19.
not ok 9 - astyle is a file

#   Failed test 'astyle is a file'
#   at t\02_binary_executable.t line 20.
not ok 10 - astyle is executable

#   Failed test 'astyle is executable'
#   at t\02_binary_executable.t line 21.
# Looks like you failed 4 tests of 10.
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/10 subtests

Test Summary Report
-------------------
t\02_binary_executable.t (Wstat: 1024 Tests: 10 Failed: 4)
  Failed tests:  7-10
  Non-zero exit status: 4
Files=1, Tests=10,  1 wallclock secs ( 0.06 usr +  0.00 sys =  0.06 CPU)
Result: FAIL

N:\home\ollisg\dev\Alien-astyle>perl -Mblib -MAlien::astyle -E "say Alien::astyle->bin_dir"
N:\home\ollisg\dev\Alien-astyle\blib\lib\auto\share\dist\Alien-astyle\bin

N:\home\ollisg\dev\Alien-astyle>dir blib\lib\auto\share\dist\Alien-astyle\bin
total 408
-rwxrwxrwx  1 ollisg 0 416768 2017-01-09 03:22 astyle.exe

N:\home\ollisg\dev\Alien-astyle>blib\lib\auto\share\dist\Alien-astyle\bin\astyle.exe --version
Artistic Style Version 2.06
```

**Comment by wbraswell** [comment]
@plicease 
I think this should all be fixed now with the latest Alien::astyle v0.006 currently on CPAN, can you please confirm?

**Comment by plicease** [comment]
works:

```
Done
Building Alien-astyle
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\Alien\astyle.pm
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\Alien\astyle\ConfigData.pm
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\Alien\astyle\Install\Files.pm
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\README
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\bin\astyle.exe
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\share\doc\astyle\html\astyle.html
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\share\doc\astyle\html\install.html
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\share\doc\astyle\html\news.html
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\share\doc\astyle\html\notes.html
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\auto\share\dist\Alien-astyle\share\doc\astyle\html\styles.css
OK
Successfully installed Alien-astyle-0.006000
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\MSWin32-x86-multi-thread-64int\.meta\Alien-astyle-0.006000\install.json
Installing N:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5\MSWin32-x86-multi-thread-64int\.meta\Alien-astyle-0.006000\MYMETA.json
1 distribution installed

N:\home\ollisg>
```
